Documentation/Signoff/Logic-gate-subsystem.md

```diff
@@ -0,0 +1,93 @@
+# Logic Gates Subsystem
+
+##  Function of the System
+The function of this subsystem is to introduce the concept of logic gates through an approachable way for someone with little to no experience using AND,OR and NOT gate. The subsystem will be user-friendly by using switches and a single LED to demonstrate these concepts and allow the user to interact with the inputs. The car will be able to be set into mode through the primary screen with the two modes being AND and OR. The two modes will move the car forward and when the user flip the Not gate switch it will invert the ouput of either mode making the car go backward.
+
+
+## Constraint
+
+| Description | Constraints                                                   | Origin            |
+| --- | --------------------------------------------------------------------- | -----------------|
+| Current regulation | Current through the LED shall be no less than 15 mA [3] | LED Constraint |
+| Voltage regulation | System shall use 5v from the Arduino UNO to operate | Device Constraint |
+|  Explicit  | Switches shall be labeled for a user-friendly interaction  | simplicity Constraint|
+| Cost Limit  |Any components and equipment within this subsystem should not exceed $40 dollars and the subsystems total cost should not exceed $120 to help keep the project budget in the 400-900 range| Conceptual Design|
+
+## Buildable schematic
+![image](https://github.com/user-attachments/assets/cff989bc-b4f0-4335-a2e1-3d9a95cced7f)
+
+
+
+
+
+The above image is a schematic of 3 switches, a current limiting resitor and a LED connected to digital port 10, 11, 12, and 13 of the Close Control Loop arduino subsystem, responsible for moving the car. Switch 1 and 2 can be either AND or OR depending on the mode. They are respectively connected to digital port 11 and 12 of the arduino. Switch 3 is the inverter, inverting either AND or OR gate depending on the mode selcted. It is connected to digital pin 10 of the arduino. The LED will serve as an output indicating when the car is in movement, connected to digital pin 13 of the arduino.
+
+
+# Analysis
+If we are analyzing the current constraint, it shall not be less thant 15 mA and no more than 30 mA as it is specified in the datasheet [1]. Excessive current can lead to component frying or malfunctionning the reason why we have this constraint in place. Since the arduino is operating on 5v [2], our subsystem will need a current limiting resistor to limit the current throught the LEDs. V = RI => R = $\frac{v}{I}$.     
+V = 5V , I = 15 mA.   
+
+R = $\frac{5}{0.015}$ = 334 ohms. To keep the current above 15mA, a 300 ohm resistor is perfect because it brings the current to about 16mA with a voltage of 5V.
```

Comment:
The voltage over the resistor is the source voltage minus the diode drop. Did you forget to include the diode drop in your calculations?